src/LuaTransformer.ts

```diff
@@ -63,11 +63,14 @@ export class LuaTransformer {
     private symbolIds: Map<ts.Symbol, tstl.SymbolId>;
     private genSymbolIdCounter: number;
 
+    private program: ts.Program;
```

Comment:
Can you make this protected and put it next to the other constructor variables (below `options`)